apps/website/src/components/content/LinkHover.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import { type AmbassadorKey } from "@alveusgg/data/src/ambassadors/core";
+import { type EnclosureKey } from "@alveusgg/data/src/enclosures";
+import DynamicCard from "./DynamicCard";
+
+import Link from "./Link";
+
+type LinkHoverProps = {
+  href: string;
+  name: string;
+  ambassador?: AmbassadorKey;
+  enclosure?: EnclosureKey;
+  staffMember?: string;
+  cardType: "ambassador" | "staff" | "enclosure";
+  yellow?: boolean;
+};
+
+const LinkHover: React.FC<LinkHoverProps> = ({
+  href,
+  name,
+  ambassador,
+  enclosure,
+  yellow,
+  cardType,
+}) => {
+  const [cardShown, setCardShown] = useState<boolean>(false);
+  const [delay, setDelay] = useState<number | null>(null);
+  const [linkPosition, setLinkPosition] = useState<DOMRect | undefined>();
+
+  const onEnter = (e: React.MouseEvent<HTMLDivElement>) => {
+    setLinkPosition(e.currentTarget.getBoundingClientRect());
+    setDelay(
+      window.setTimeout(() => {
+        setCardShown(true);
+      }, 500)
+    );
+  };
+  const onLeave = () => {
+    setCardShown(false);
+    if (delay) window.clearTimeout(delay);
+  };
+  return (
+    <div onMouseEnter={onEnter} onMouseLeave={onLeave} className="inline">
+      {cardShown && (
+        <DynamicCard
+          cardType={cardType}
+          linkPosition={linkPosition && linkPosition}
+          ambassador={ambassador}
+          enclosure={enclosure}
+          staffMember={cardType === "staff" ? name : ""}
+        />
+      )}
+      <Link className={yellow ? "text-yellow-400" : ""} href={href}>
```

Comment:
Absolutely, I'll stick to the standard Link colors.
Added that change because it seemed like it made it a little more difficult to read in this background, but could just be me:
![image](https://github.com/alveusgg/alveusgg/assets/73081185/7daccbc0-ae5d-4318-a6d2-2a56d7b7785d)

Reply:
There is a `dark` prop on `Link` that should fix that iirc